.github/pull_request_template.md

```diff
@@ -0,0 +1,38 @@
+## Changes
+
+_Explanation of changes goes here_
+
+## Notes
+
+_Any other notes go here_
+
+## To Do
+
+_Any remaining things that need to get done_
+
+- [ ] item 1
+- [ ] ...
+
+## Checklist
+
+It can be helpful to check the `Checks` and `Files changed` tabs.
+Please review the [contributor guide](https://github.com/Northeastern-Electric-Racing/PM-Dashboard-v2/blob/main/docs/ContributorGuide.md) and reach out to your squad if anything is unclear.
+
+- [ ] All commits are tagged with the ticket number
+- [ ] No linting errors
+- [ ] No newline at end of file warnings
+- [ ] All code follows repository-configured prettier formatting
+- [ ] No merge conflicts
+- [ ] All checks passing
+- [ ] Code sufficiently tested
```

Comment:
Can we turn this into its own "Test Cases" section? This would help make it clear to the reviewer what was done to test the changes made, and if no tests were necessary the author can just put "None" or something like that.